messages/store.json

```
@@ -118,6 +118,34 @@
             "done": "Done"
         }
     },
+    "extension": {
+        "register": {
+            "cmdDescription": "Register an extension with one command",
+            "apexClassNameFlagDescription": "Name of the Apex class that needs to be extended",
+            "errApexClass": "Error while retrieving the Apex Class:",
+            "error": "Error while registering with the given registered extension name:",
+            "extensionPointNameDescription": "Means of identifying entry points within the domain code that are capable of being extended by partners.Takes the form domain_sub-domain_interface. E.g: checkout_summary_computeTax",
+            "regExtensionNameFlagDescription": "Unique name for extension",
+            "errEPN": "Invalid EPN. Run [commerce:extension:points:list] for all valid EPN values",
+            "savingConfigIntoConfig": "Sucessfully registered Apex Class. Use command [commerce:extension:map] to link the extension to a store.",
```

Should we state that the extension was registered rather than the Apex class. In the future we plan to support other forms of extensions
```suggestion
            "savingConfigIntoConfig": "Sucessfully registered the extension Use command [commerce:extension:map] to link the extension to a store.",
```